benchmark/v8/get-stats.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const common = require('../common.js');
+const v8 = require('v8');
+
+const bench = common.createBenchmark(main, {
+  method: [
+    'getHeapStatistics',
+    'getHeapSpaceStatistics'
+  ],
+  n: [1e6],
+  flags: ['--ignition --turbo', '']
```

Comment:
Setting the flags like this runs the benchmark with two separate configurations, one with `--ignition --turbo` and the other without flags. I wasn't sure myself but this definitely worked when I ran it like this :-)